Editor/Editors/CinemachineVirtualCameraBaseEditor.cs

```diff
@@ -143,8 +146,41 @@ protected void DrawHeaderInInspector()
             {
                 DrawCameraStatusInInspector();
                 DrawGlobalControlsInInspector();
+                ExcludeProperty("Header");
             }
-            ExcludeProperty("Header");
+        }
+        
+        static GUIContent s_InputProviderAddLabel = new GUIContent(
+            "Add Input Provider", "Adds CinemachineInputProvider component to this vcam, if it does not have one, " +
+            "enabling the vcam to read input from Input Actions. By default, a simple mouse XY input action is added.");
+        /// <summary>
+        /// Draws a button to the inspector that can add CinemachineInputProvider component to the virtual camera,
+        /// if the InputSystem package is installed, and the virtual camera does not already have this button,
+        /// and requires input, and does not have CinemachineInputProvider component.
+        /// </summary>
+        /// <param name="vcamBase">The virtual camera</param>
+        protected void DrawInputProviderButton(CinemachineVirtualCameraBase vcamBase)
```

Comment:
don't pass the target as a param, it suggests that this can be used for any vcam.  It can't.  It's good for this inspector's target only.